sessionctx/variable/session.go

```diff
@@ -97,40 +97,40 @@ func (tc *TransactionContext) UpdateDeltaForTable(tableID int64, delta int64, co
 
 // SessionVars is to handle user-defined or global variables in current session.
 type SessionVars struct {
-	// user-defined variables
+	// Users variables
 	Users map[string]string
-	// system variables
+	// Systems variables
 	Systems map[string]string
-	// prepared statement
+	// PreparedStmts store prepared statement.
 	PreparedStmts        map[uint32]interface{}
 	PreparedStmtNameToID map[string]uint32
-	// prepared statement auto increment id
+	// preparedStmtID is id of prepared statement.
 	preparedStmtID uint32
 
 	// retry information
 	RetryInfo *RetryInfo
 	// Should be reset on transaction finished.
 	TxnCtx *TransactionContext
 
-	// following variables are special for current session
+	// Status uint16 // following variables are special for current session
```

Comment:
// following variables are special for current session
// Status is the status of the current session.